staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1243,7 +1243,9 @@ type ManagedFieldsEntry struct {
 	// APIVersion field. It is necessary to track the version of a field
 	// set because it cannot be automatically converted.
 	APIVersion string `json:"apiVersion,omitempty" protobuf:"bytes,3,opt,name=apiVersion"`
-	// Time is timestamp of when these fields were set. It should always be empty if Operation is 'Apply'
+	// Time is the timestamp of when these fields were set. The timestamp will
+	// also be updated if any of the owned fields value changes (even if the
+	// ownership doesn't change).
```

Comment:
> Is it updated in these scenarios:
> 
> * A field is added to this entry

Yes

> * A field is removed from this entry because this manager removed it (I think so?)

No

> * A field is removed from this entry because another manager took it over (I'm not sure?)

No

> * The value of an existing field in this entry is changed by this manager

Yes

Reply:
Basically, we only update the timestamp of the entry we're currently updating/creating. So, if A takes something from B's entry, we're not updating the timestamp of B's entry, we're only updating the timestamp for A's new entry.

Reply:
```diff
apelisse edited 5 minutes ago
| A field is removed from this entry because this manager removed it (I think so?)
- No
+ Yes
```

whew... I was afraid I *really* misunderstood this

Reply:
> Basically, we only update the timestamp of the entry we're currently updating/creating. So, if A takes something from B's entry, we're not updating the timestamp of B's entry, we're only updating the timestamp for A's new entry.

Even though we actually do update B's entry to remove the field A is taking over

Reply:
Correct

Reply:
ok... would it be helpful to call out those specific examples in the doc?

Reply:
Please also consider documenting some of this in https://kubernetes.io/docs/reference/using-api/server-side-apply/, or having that page hyperlink to this part of the API reference information.

Reply:
I've updated the doc string, PTAL!